mirrord/agent/src/steal/http.rs

```diff
@@ -20,7 +20,7 @@ pub(super) type DefaultReversibleStream = ReversibleStream<MINIMAL_HEADER_SIZE>;
 
 /// Identifies a message as being HTTP or not.
 #[derive(Debug, Default, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
-enum HttpVersion {
+enum HttpVersionChecker {
```

Comment:
Why the rename?

Reply:
Artifact from when I was trying out stuff and had a bunch of names looking similarly to `HttpVersionX`.

Reverted.